### ISSUE TITLE
Review workflows

### ISSUE THREAD

**Issue by aablsk** [gitbutlerapp/gitbutler#2924]
I work in a trunk-based development environment, which often means doing reviews post-merge. 

I'd love it if Gitbutler supported a review flow that supports reviewing a selection of specific commits.

**Comment by polisherm** [comment]
I strongly agree with you. In cuurrent flow, reviewing other co-worker's branch is difficult.

**Comment by krlvi** [comment]
Thanks for this suggestion. This is something that we are starting to work on (and it's not a small feature, so it will take a while). Hopefully it won't be too long until we have something that we can share
